assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/pentahoServices.spring.xml

```diff
@@ -134,8 +128,8 @@
   </bean>
   <bean class="org.pentaho.platform.web.http.api.resources.EmailResource" scope="request"/>
   <bean class="org.pentaho.platform.web.http.api.resources.SessionResource" scope="request"/>
-  <bean class="org.pentaho.platform.web.http.api.resources.SchedulerResource" scope="request"/>
```

Comment:
We need to remove this code not comment it

Reply:
Rmoved this commented code @rmansoor